### ISSUE TITLE
Framework for trying out variety of Keras architectures (python)

### ISSUE THREAD

**Issue by vincentvanhees** [NLeSC/mcfly#18]
Input:
- List with compiled model objects (predefined architectures and hyperparameters in keras)
- data, start with UCR data

Output:
- Performance per model object (bv accuracy, plots, sklearn, ....)
- Trained weights per model object that can be used for unseen data classification

**Comment by dafnevk** [comment]
I'll add some functionality to actually loop over different models and compare performamce between architectures.
